interceptor/middleware/loggingresponsewriter.go

```diff
@@ -46,3 +49,12 @@ func (lrw *loggingResponseWriter) WriteHeader(statusCode int) {
 
 	lrw.statusCode = statusCode
 }
+
+// implements http.hijacker
```

Comment:
Small change to force `loggingResponseWriter` to implement `http.Hijacker`

```suggestion
var _ http.Hijacker = (*loggingResponseWriter)(nil)

```